.tool-versions

```diff
@@ -1,2 +1,2 @@
 nodejs 12.13.1
-ruby 2.6.2
+ruby 2.7.2
```

Comment:
Nothing against, but any reason for this upgrade?

Reply:
forgot to separate this, actually. but it was purely for convenience. I didn't have 2.6 installed, and this was easier than installing it along with all the image_optim stuff :stuck_out_tongue: 

I assumed it was no big deal since everyone will eventually move to more recent versions anyway, and will end up with similar nuisances

Reply:
It's no big deal, just found it weird this happened in a post PR.